content/en/serverless/installation/nodejs.md

```diff
@@ -59,10 +59,11 @@ yarn global add @datadog/datadog-ci
 
 ### Configure credentials
 
-For a quick start, configure Datadog and [AWS credentials][1] using the following command. For production applications, consider supplying the environment variables or credentials in a more secure manner. 
+For a quick start, configure Datadog and [AWS credentials][1] using the following command. For production applications, consider supplying the environment variables or credentials in a more secure manner.
 
 ```bash
 export DATADOG_API_KEY="<DD_API_KEY>"
+export DATADOG_SITE="<DD_SITE>" # eg datadoghq.com, datadoghq.eu, etc
```

Comment:
```suggestion
export DATADOG_SITE="<DD_SITE>" # such as datadoghq.com or datadoghq.eu
```

Reply:
@DarcyRaynerDD let's list out all the possible values: `datadoghq.com`, `datadoghq.eu`, `us3.datadoghq.com` and `ddog-gov.com`. It's sometimes very difficult for the customer to figure out the right full site name, since they are not named very consistently.